msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -286,7 +286,7 @@ public Boolean getMultipleCloudsSupported() {
      * @return The boolean indicator of whether or not to use the broker.
      */
     public Boolean getUseBroker() {
-        return mUseBroker;
+        return mUseBroker == null ? true : false;
```

Comment:
```suggestion
        return (mUseBroker == null) || mUseBroker;
```

Reply:
Suggested a change above, looks like we are ignoring the supplied value if it is not null

Reply:
You're right.  I wrote this while not paying attention.